app/angular/package.json

```diff
@@ -38,26 +35,19 @@
     "babel-runtime": "^6.23.0",
     "case-sensitive-paths-webpack-plugin": "^2.1.2",
     "core-js": "^2.5.5",
-    "cross-env": "^5.1.4",
```

Comment:
So this : `"dev": "cross-env DEV_BUILD=1 nodemon -e js,ts --watch ./src --exec \"yarn prepare\"",` is not needed anymore (I think it wasn't in use before anyway) ?

Reply:
It's executed in root, so no need to have those dependencies on this level

Reply:
Or maybe it's not, then we can remove the script as well

Reply:
I think it's not (maybe it was, but a long time ago). 
The last time I've been touching there was due to the ts distribution issue, and I've cleaned thing there. 
But maybe I am missing something

Reply:
Oh, I am late